plugin/src/init.server.lua

```diff
@@ -20,6 +20,11 @@ local app = Roact.createElement(App, {
 })
 local tree = Roact.mount(app, nil, "Rojo UI")
 
+game.Close:Connect(function()
```

Comment:
`game.Close` doesn't exist from my search, do you mean `OnClose` (deprecated) or `BindToClose`?

Reply:
i guess its undocumented?
![image](https://user-images.githubusercontent.com/9562836/129132085-5cb1eda7-d3a8-4b35-9881-e6b3bd318703.png)

Reply:
nope, its https://developer.roblox.com/en-us/api-reference/event/ServiceProvider/Close

Reply:
Whack, wonder why it's not on Anaminus' site that I checked.

Reply:
am i okay to mark this as resolved?

Reply:
Is there a reason we're using this instead of `BindToClose` ? As in, is there some behaviour we need from this obscure method. I think we should understand this thing before we put it into Rojo.

Reply:
nah, ill just change it to BindToClose